code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -376,6 +376,19 @@
 	M.radiation += 4
 	..()
 
+/datum/reagent/toxin/magillitis
```

Comment:
Just to make it clear that it's a bad chemical. Can move it anywhere if requested.